indicators/no_critical_vulnerability.json

```diff
@@ -0,0 +1,24 @@
+{
+    "@context": "https://w3id.org/everse/rsqi#",
+    "name": "noCriticalVulnerability",
+    "description": "Checks if reported critical vulnerabilities has been fixed",
+    "keywords": ["security", "critical", "vulnerability"],
+    "identifier": { "@id": "https://w3id.org/everse/i/indicators/no_critical_vulnerability"},
+    "status": "Active",
+    "version": "1.0.0",
+    "author": {
+        "@type": "schema:Organisation",
+        "name": "OpenSSF"
+    },
+    "source": {
+        "@id": "https://www.bestpractices.dev/en/criteria/0#0.vulnerabilities_critical_fixed",
+        "name": "OpenSSF Best practice Critical Vulnerability Fixed"
+    },
+    "contact": {
+        "@type": "schema:Person",
+        "name": "Tom François",
+        "schema:email": "tom.francois@lapp.in2p3.fr"
+    },
+    "qualityDimension": {"@id": "https://w3id.org/everse/i/dimensions/security"},
+    "created": "03-04-2025"
```

Comment:
This indicator was not created in 2025, no? Do we know when was it created?
It may be up for discussion if created is the date of addition in everse, or the creation of the indicator. I would interpret it the same as author (author of the indicator). If not known, I would not add it